### PR TITLE
Add lots of tooltips to help explain stuff.

### PR DESCRIPTION
**Pull request by ralphbean**

### CODE REVIEW

File: fmn/web/templates/context.html [hunk 1]
```diff
@@ -107,23 +107,40 @@ <h4 class="list-group-item-heading">{{fltr.name}}</h4>
...
+            btn-success" data-toggle="tooltip" title="This filter is not
+            currently enabled.  Use this button to re-enable it.">
               <span class="glyphicon glyphicon-ok"></span>
               Enable this filter</button>
             {% endif %}
-            <button type="submit" name="method" value="delete" class="btn btn-danger">
+            <button type="submit" name="method" value="delete" class="btn
+            btn-danger" data-toggle="tooltip" title="WARNING - Clicking this
+            button will permanently delete this filter.  Consider disabling it
+            instead.">
```
Comment (pypingou) [inline fmn/web/templates/context.html]: Should we use a js pop-up with yes/no instead of a simple warning in the title?

File: fmn/web/templates/filter.html [hunk 2]
```diff
@@ -15,27 +15,36 @@
       <input name="filter_name" id="filter_name" value="{{filter.name}}" type="hidden">
       <div class="btn-group pull-right">
         <span>
-        <a class="btn btn-info" href="{{url_for('context_json', openid=openid, context=current)}}">
+        <a class="btn btn-info" href="{{url_for('context_json', openid=openid, context=current)}}"
+          data-toggle="tooltip" title="Export as JSON">
```
Comment (pypingou) [inline fmn/web/templates/filter.html]: Export what maybe?

**Comment by pypingou** [comment]
2 questions otherwise :+1:

File: fmn/web/templates/context.html (hunk 1 above)
Comment (ralphbean) [inline fmn/web/templates/context.html]: Sure, but we should file an issue for it and make it another PR.  That'll take a bit more work.
